internal/configs/parsing_helpers.go

```diff
@@ -183,7 +183,7 @@ func ParseUint64(s string) (uint64, error) {
 }
 
 // timeRegexp http://nginx.org/en/docs/syntax.html
-var timeRegexp = regexp.MustCompile(`^([0-9]+([ms|s|m|h|d|w|M|y]?){0,1} *)+$`)
```

Comment:
just noticed another bug. it was present long time ago :( 

value like "5s 5s" are considered valid, which can make NGINX fail to reload:

```
  upstreams:
  - connect-timeout: "5s 5s"
    name: tea
    port: 80
    service: tea-svc
```

```
"nginx: [emerg] invalid number of arguments in \"proxy_connect_timeout\" directive in /etc/nginx/conf.d/vs_default_cafe.conf:164\n"
```

Reply:
Fixed that, but I also think "11 11" is invalid, so I've changed the test data to reflect this, let me know what you think.